docs/blog/2019-10-17-how-gatsby-can-power-your-site-offline-even-without-javascript/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "How Gatsby can power your site offline - even without JavaScript"
+date: 2019-10-17
```

Comment:
We'll want to update the date here and in the filename so this gets merged at the top of the queue!